src/components/reporter/Viewers/CReportMetric.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div
```

Comment:
IIRC this is a copy-paste of what we've done for Compose metric page block; can we move all this to a new component?

Reply:
Yes we can but i would wait for the big merge of 2022.9 where charts/metric will be properly merged into reusable components and classes